.github/workflows/avm.platform.manage-issue-for-failing-pipelines.yml

```
@@ -0,0 +1,35 @@
+name: ".Platform: Manage issues for failing pipelines"
```

Please name after the file name. E.g. after renaming the yml file, depending on the name chosen, replace with something like
```suggestion
name: "avm.platform.manage-workflow-issue"
```